internal/rest/avi_obj_httpps.go

```diff
@@ -120,9 +121,22 @@ func (rest *RestOperations) AviHttpPSBuild(hps_meta *nodes.AviHttpPolicySetNode,
 		}
 		match_target := avimodels.MatchTarget{}
 
-		if len(hppmap.Path) > 0 {
+		if hppmap.StringGroupRefs != nil && len(hppmap.StringGroupRefs) > 0 {
+			match_crit := hppmap.MatchCriteria
+			var match_case string
+			if hppmap.MatchCase != "" {
+				match_case = hppmap.MatchCase
+			} else {
+				match_case = "SENSITIVE"
```

Comment:
Why is it that match_case is `SENSITIVE` whereas the documentation says it should be insensitive https://confluence.eng.vmware.com/pages/viewpage.action?spaceKey=NSBU&title=Support+for+NGINX+annotation+equivalents+in+AKO+-+Phase+1#:~:text=case%20will%20be-,INSENSITIVE,-.%20If%20the%20useRegex

Reply:
Also if SENSITIVE is being used at multiple places, why not define ince and use everywhere ?

Reply:
If this field **hppmap.MatchCase** then we are taking the value as default **SENSITIVE**. In regex case, this field will always be set as **INSENSITIVE** in graph layer.